benchmark.py

```diff
@@ -33,9 +33,10 @@ def time_run(cmdline):
     ('AWK', 'gawk -f simple.awk', 'mawk -f optimized.awk', 'optimized uses `mawk`'),
     ('Forth', '../gforth/gforth-fast simple.fs', '../gforth/gforth-fast optimized.fs', ''),
     ('Shell', 'bash simple.sh', 'bash optimized.sh', 'optimized does `LC_ALL=C sort -S 2G`'),
+    ('Julia', 'julia simple.jl', None, 'by Alessandro Melis'),
```

Comment:
Note that this means that your timing includes startup time, which in the case of Julia includes JIT compilation.   Since this problem is so small (my [optimized version](https://discourse.julialang.org/t/count-words-challenge/57221/16?u=stevengj) runs in 80ms on my machine), the compilation overhead is significant.

(For long-running performance-critical problems, typically Julia is used by launching `julia` once and then running any loops inside Julia, so that one-time compilation overheads don't matter, as opposed to launching `julia` with a little script that runs in a fraction of a second many times.)